easybuild/easyconfigs/l/libplinkio/libplinkio-0.9.8-GCCcore-9.3.0.eb

```
@@ -0,0 +1,31 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'CMakeMake'
+
+name = 'libplinkio'
+version = '0.9.8'
+
+homepage = "https://github.com/mfranberg/libplinkio"
+description = """A small C and Python library for reading PLINK genotype files."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'cstd': 'gnu17', 'extra_cflags': '-w'}
+
+github_account = "mfranberg"
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/archive/refs/tags']
```

Since you're already setting `github_account`, you can make use of the template variable `GITHUB_SOURCE` (see https://docs.easybuild.io/en/latest/version-specific/easyconfig_templates.html#template-constants-that-can-be-used-in-easyconfigs):
```suggestion
source_urls = [GITHUB_SOURCE]
```

---

Other than that, this looks fine. So if you could make this final change, I'll start some test builds and merge the PR.

---

As it's a trivial change, I'll just commit this.